pyqtgraph/colormap.py

```diff
@@ -47,16 +55,25 @@ def listMaps(source=None):
 
 def get(name, source=None, skipCache=False):
     """
-    Warning, highly experimental, subject to change.
-
-    Returns a ColorMap object from a local definition or imported from another library
-    ===============  =================================================================
-    **Arguments:**
-    name             Name of color map. Can be a path to a defining file.
-    source           'matplotlib' imports a map defined by Matplotlib
-                     'colorcet' imports a maps defined by ColorCET
-                     otherwise local data is used
-    ===============  =================================================================
+    .. warning:: Experimental, subject to change.
+
+    Returns a ColorMap object from a local definition or imported from another library.
+    The generated ColorMap objects are cached for fast repeated access.
+    
+    Parameters
+    ----------
+    name: str
+        Name of color map. In addition to the included maps, this can also 
+        be a path to a file in the local folder. See the files in the
+        `pyqtgraph/colors/maps/` folder for examples of the format.
```

Comment:
Double backticks for RST, annoying I know

Reply:
⇒ changed.